provisioners/deploy-backend.yml

```diff
@@ -38,6 +38,10 @@
         - task-runner
         - upload-service
         - website
+    - name: Install upload-service dependencies
+      community.general.npm:
```

Comment:
I'm not sure we have npm in the path on prod-backend (when I run `which npm` I get nothing).  It's not in `usr/local/bin`, where this package expects it, at least.  I see that we use it in the [configuration script](https://github.com/PermanentOrg/infrastructure/blob/main/provisioners/configure.sh#L70) so presumably it's there somewhere, but can you confirm?

Reply:
```
root@dev:/home/aatwood# npm -v
6.14.11
```

That's what I get as root!

Reply:
Oh you know what, I'm silly - was looking at prod, where the new changes don't exist yet.  Sorry about that!